.github/workflows/ci.yml

```diff
@@ -0,0 +1,163 @@
+          # https://github.com/actions/virtual-environments/issues/34
+          # Therefore, ujson can't be compiled and the build would fail.
+          - os: windows-latest
+            python-version: 3.5
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install latest nightly
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          override: true
+
+      - name: Lint with rustfmt
+        uses: actions-rs/cargo@v1
+        with:
+          command: fmt
+
+      - name: Lint with clippy
+        uses: actions-rs/cargo@v1
+        with:
+          command: clippy
+          args: --all-targets --all-features
+
+      - name: Test with cargo
+        uses: actions-rs/cargo@v1.0.1
+        with:
+          command: test
+          toolchain: nightly
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install poetry
+          poetry install
+
+      - name: Build Python package
+        run: poetry run maturin develop --release
+
+      - name: pytest
+        run: poetry run pytest tests
+
+      - name: Benchmark
+        run: poetry run pytest benchmarks --compare
+
+  publish-rust:
+    if: startsWith(github.ref, 'refs/tags/')
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          override: true
+
+      - name: Publish on crates.io
+        run: |
+          cargo login ${{ secrets.CARGO }}
+          cargo publish
+
+  publish-python:
+    needs: build
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        exclude:
+          # There is a known issue where Python C extensions
+          # can not be built on Python 3.5 on Windows.
+          # https://github.com/actions/virtual-environments/issues/34
+          # Therefore, ujson can't be compiled and the build would fail.
+          - os: windows-latest
+            python-version: 3.5
+    steps:
+      - uses: actions/checkout@v1
+
+      - uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install latest nightly
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          override: true
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install poetry
+          poetry install
+
+      - name: Build Python package
+        run: poetry run maturin build --release --no-sdist --strip --interpreter python${{matrix.python_version}}
```

Comment:
Is there a reason for not using publish here directly?

Reply:
The idea was to do a kind of "integration test" and install the wheels locally before publishing them. Do you think that makes sense?

Reply:
My idea was that people use e.g. tox with the PEP 517 backend for testing (which unfortunately still [doesn't work](https://github.com/tox-dev/tox/issues/1344)), but given that installing the wheels locally is likely the most reasonable thing to do.